sklearn/cluster/_spectral.py

```diff
@@ -370,7 +370,8 @@ class SpectralClustering(ClusterMixin, BaseEstimator):
         callable object. Ignored by other kernels.
 
     n_jobs : int or None, optional (default=None)
-        The number of parallel jobs to run.
+        The number of parallel jobs to run for neighbors search in
+        constructing the affinity matrix.
```

Comment:
Looks like this parameter is only used when `affinity` is `nearest_neighbors` or `precomputed_nearest_neighbors`.